contrib/libvirt/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "libvirt"
+pkgver = "10.0.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "libxml2-progs",
+    "meson",
+    "perl",
+    "pkgconf",
+    "python-docutils",
+    "xsltproc",
+]
+makedepends = [
+    "glib-devel",
+    "gnutls-devel",
+    "libtirpc-devel",
+    "libxml2-devel",
+    "linux-headers",
+]
+pkgdesc = "API, daemon, and management tool for virtualization"
+maintainer = "Nova <froggo8311@proton.me>"
+license = "GPL-2.0-only"
+url = "https://libvirt.org"
+source = f"https://download.libvirt.org/libvirt-{pkgver}.tar.xz"
+sha256 = "8ba2e72ec8bdd2418554a1474c42c35704c30174b7611eaf9a16544b71bcf00a"
+# virsh-optparse fails
+options = ["!check"]
```

Comment:
fix it or skip the 1 test instead of all of them then

Reply:
figured it out- because of the above and forgetting every library, it's missing yajl for json support so

```
root@cbuild: /builddir/libvirt-10.0.0$ ./build/tools/virsh -q -c test:///default qemu-monitor-command test a
error: internal error: No JSON parser implementation is available
```

instead of the expected result

adding yajl alone runs like double the tests even

Reply:
not having yajl even prevented qemu support from being built (disabled in log above, enabled after adding it), meaning this wasn't even usable to begin with

Reply:
hm, seems like it just didnt like me or something. i re-ran without check disabled and it appears to be fine